pkgs/development/python-modules/gst-python/default.nix

```diff
@@ -1,10 +1,10 @@
-{ buildPythonPackage, fetchurl, stdenv, meson, ninja, pkgconfig, python, pygobject3
+{ buildPythonPackage, fetchurl, stdenv, meson, ninja, pkgconfig, python, python3, pygobject3
```

Comment:
The `python` here is the Python used for the package set. Does the `python2` build also require `python3`?

Reply:
Yes, it does.

Reply:
Actually, it looks like it only supports Python 3 now: https://github.com/GStreamer/gst-python/blob/master/meson.build#L22